src/store/models/tokenCreation.ts

```diff
@@ -0,0 +1,37 @@
+import { createModel } from '@rematch/core';
+
```

Comment:
I dont think we need a separate model for this as token transaction is just a usual TKL transaction, we can just use currentTransaction model. 

We use currentTransaction for token transactions.